sdks/python/apache_beam/testing/data/trigger_transcripts.yaml

```diff
@@ -118,6 +139,7 @@ broken_on:
   - SwitchingDirectRunner
 window_fn: Sessions(10)
 trigger_fn: AfterWatermark(early=AfterCount(2), late=AfterCount(3))
+allowed_lateness: 85
```

Comment:
Nit: 85 seems an odd value. An even 100 would be better.

Reply:
Done.